### ISSUE TITLE
Search not working on mobile

### ISSUE THREAD

**Issue by tttapa** [doxygen/doxygen#8712]
**Describe the bug**
When tapping the search field in a mobile browser, it immediately loses focus and it is impossible to type a query.

**Screenshots**
In the following recordings, I'm just tapping the search field:

1.8.20 works fine (Chrome):

https://user-images.githubusercontent.com/21296729/129123600-6d1c8d6a-6195-48cb-813b-9548cf608e1b.mp4

1.9.2 (master) doesn't work (Chrome):

https://user-images.githubusercontent.com/21296729/129123611-005acad6-eae2-434c-835e-91ef69307a82.mp4

1.9.2 (master) doesn't work (Firefox):

https://user-images.githubusercontent.com/21296729/129123619-8dfb1b84-8e8f-4cbd-9042-6f868636b18d.mp4

**To Reproduce**  
```sh
cat > header.h << EOF
/** @brief Some class. */
class C {};
EOF
doxygen -g
doxygen
```
[**Doxygen-8712.zip**](https://github.com/doxygen/doxygen/files/6972179/Doxygen-8712.zip)

**Version**  
1.9.2 (a4958eb7f24b72062a07439235fa9345e81836a7), current master version, compiled from source just now. Ubuntu 20.04 x86_64.

**Comment by doxygen** [comment]
@tttpa: I've tried on iOS (iPhone) and there the search results window appears ok (a bit too big but that's a different issue).
Can you see if you get any error on the Javascript console? with chrome you can open a new tab and go to URL `chrome://inspect` to toggle logging on and off.

**Comment by tttapa** [comment]
It does indeed seem to work fine on iOS (both iPhone and iPad).  
I tried a different Android phone, and it has the same issues as in the videos above.

I used `chrome://inspect`, and there are these messages:

```text
[Intervention] Images loaded lazily and replaced with placeholders. Load events are deferred. See https://crbug.com/954323
```
![Screenshot from 2021-08-14 21-50-32](https://user-images.githubusercontent.com/21296729/129458614-b6e94dd5-df45-46a3-9951-ab2a1973d24e.png)

**Comment by doxygen** [comment]
@tttapa Please verify if the problem is solved with the referenced commit. It is a bit hard for me to test as I don't own an Android mobile device. Only tested with the simulator that comes with Android Studio.

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.2.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).

**Comment by tttapa** [comment]
> Please verify if the problem is solved with the referenced commit. It is a bit hard for me to test as I don't own an Android mobile device. Only tested with the simulator that comes with Android Studio.

That does indeed seem to fix it, thanks a lot!
